src/main/java/org/stellar/sdk/Address.java

```diff
@@ -0,0 +1,170 @@
+package org.stellar.sdk;
+
+import com.google.common.base.Objects;
+import org.stellar.sdk.xdr.Hash;
+import org.stellar.sdk.xdr.SCAddress;
+import org.stellar.sdk.xdr.SCVal;
+import org.stellar.sdk.xdr.SCValType;
+
+/**
+ * Represents a single address in the Stellar network. An address can represent an account or a
+ * contract.
+ */
+public class Address {
+
+  private final byte[] key;
+
+  private final AddressType type;
+
+  /**
+   * Creates a new {@link Address} from a Stellar public key or contract ID.
+   *
+   * @param address the Stellar public key or contract ID.
```

Comment:
```suggestion
   * @param address the StrKey encoded format of Stellar public key or contract ID/
```

Reply:
Fixed in 0a473e72d4d0f446bc8b2f399b623096b1f599fa